Library/Formula/libfreenect.rb

```diff
@@ -28,6 +29,10 @@ def install
       args << "-DCMAKE_OSX_ARCHITECTURES=#{Hardware::CPU.universal_archs.as_cmake_arch_flags}"
     end
 
+    if build.with? "openni2"
```

Comment:
Since there's not multiple arguments to pass with this option let's just do `args << "-DBUILD_OPENNI2_DRIVER=ON" if build.with? "openni2"`, Thanks!